src/assets/scripts/client/navigationLibrary/NavigationLibrary.js

```diff
@@ -64,6 +87,11 @@ class NavigationLibrary {
         return sidProcedureModels.length > 0;
     }
 
+    /**
+     *
+     * @property hasStars
+     * @return {boolean}
```

Comment:
Since this is a getter, we like to use `@property` and `@value`, instead of `@method` and `@return`. I see you caught on to the former, so just please change the `@return` to a `@value`.

Also note that we typically like to have a `@for NavigationLibrary` in there. Comes in quite handy when reading many diffs in long files. Just thought I'd point it out, but this is obviously not an issue.

Reply:
are you sure that you mean @value? I searched the whole project and there is not a single "@value". I checked other properties and they all seem to use @property and @return.

Reply:
Nope, I guess I meant `@type`. :face_with_head_bandage:

Reply:
For example, the getters in `Fms.js`:
```javascript
    /**
     * Return the next waypoint which has an altitude restriction
     *
     * @for Fms
     * @property nextAltitudeRestrictedWaypoint
     * @type {WaypointModel}
     */
    get nextAltitudeRestrictedWaypoint() {
        const waypoints = this.getAltitudeRestrictedWaypoints();

        return waypoints[0];
    }
```

Reply:
Thanks for clarifying :+1:

Reply:
done